components/organisms/UserSettingsPage/user-settings-page.tsx

```diff
@@ -281,20 +281,15 @@ const UserSettingsPage = ({ user }: UserSettingsPageProps) => {
                     name="bio"
                     className="w-full focus:outline-none placeholder:font-normal placeholder-slate-400 bg-inherit"
                     value={bio}
+                    maxLength = {255}
                     onChange={(e) => setBio(e.target.value)}
                   ></textarea>
                 </div>
               </label>
 
-              {bio?.length > 255 ? (
-                <p aria-live="assertive" className="text-light-red-10 text-xs">
-                  Bio too long
-                </p>
-              ) : (
-                <p aria-live="polite" className="text-xs">
-                  {bio?.length}/255
-                </p>
-              )}
+              <p aria-live="polite" className="text-xs">
```

Comment:
@BekahHW do you think being unable to type further and seeing `255/255` in the UI is clear enough? If not we can put a message along the lines of max length of bio reached when they hit 255 characters. The browser won't allow anymore characters because of the `maxLength` here which is what I proposed in the good first issue linked to this PR.

Reply:
Sorry, idk how I missed this.

We could put a message there, but I think the highlight one is too much here bc the input is so small.

This is what it currently looks like when it's too long:

![image](https://github.com/open-sauced/app/assets/34313413/70c82a62-dbc0-4eb3-8352-e8438f9e4e79)

What if we keep the 255/255 and have a message that says Bio Max Length underneath?

Reply:
Did we decide on something for this @BekahHW?

Reply:
@nickytonline I think my last suggestion is the way to go.

255/255 in red
Message: Bio Max Length in red